packages/@lwc/engine-core/src/framework/restrictions.ts

```diff
@@ -76,8 +76,9 @@ export function patchElementWithRestrictions(
             get(this: Element): string {
                 return originalOuterHTMLDescriptor.get!.call(this);
             },
-            set(this: Element, _value: string) {
-                throw new TypeError(`Invalid attempt to set outerHTML on Element.`);
+            set(this: Element, value: string) {
+                logError(`Invalid attempt to set outerHTML on Element.`);
+                return originalOuterHTMLDescriptor.set!.call(this, value);
```

Comment:
It's interesting to me that some accessors work fine when overriding them, but others (as I found in #3412) throw an `Illegal invocation` error. 🤷